presto-native-execution/presto_cpp/main/PrestoToVeloxQueryConfig.cpp

```diff
@@ -159,6 +159,16 @@ void updateFromSystemConfigs(
          return folly::to<std::string>(velox::config::toCapacity(
              value, velox::config::CapacityUnit::BYTE));
        }},
+       {std::string{SystemConfig::kTaskWriterCount},
```

Comment:
Also please move these two upward to be together with the ones without custom converter.